Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,43 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Start program
+            Console.WriteLine("Hello World! Welcome to the first C# 2020 Bootcamp Homework.");
+            Program.HW1();
+            Console.WriteLine("Okay, Person #2:");
+            Program.HW1();
+        }
+        static double GetBMI(double weight, int height) //calculates the BMI by receiving weight and height as parameters and dividing weight by height squared (kg/m^2)
+        {
+            //Function Declarations
+            double BMI, hSquared;
+
+            //convert cm to meters and square
+            hSquared = (height / 100d);
+            hSquared *= hSquared;
+
+            BMI = weight / hSquared;
+
+            return BMI;
+        }
+        static void HW1()
+        {
+            //Variable Declarations
+            string name, surname;
+            int age, height; //height in cm
+            double weight; //weight in kg
```

Comment:
Because you're already using functions, I give you a slightly more advanced review feedback. First of all, I don't like the way you first declare the variables, then initialize them. I believe this is common in C++, but not in C#. Please declare and initialize your variables immediately when possible. This way you avoid default values that may potentially result in bugs.
An example:
`double height = double.Parse(Console.ReadLine());`